Backend/Remora.Discord.API.Abstractions/API/Objects/Users/IPartialUser.cs

```diff
@@ -20,6 +20,7 @@
 //  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 //
 
+using System;
```

Comment:
Added a using without other code changes?

Reply:
`[Obsolete]` attribute

Reply:
Yeah but nothing else has changed in the file, and `main` builds